tensorrt_llm/_torch/models/modeling_cohere2_vision.py

```
@@ -0,0 +1,374 @@
+            bias=True,
+        )
+        self.act = nn.SiLU()  # TODO: check the counterpart in TRT LLM
+        self.down_proj = Linear(
+            self.intermediate_size // 2,
+            config.text_config.hidden_size,
+            bias=True,
+        )
+
+    def pixel_shuffle(self, image_features: torch.Tensor): # B, S, D
+        # Concatenate a number of horizontal and vertical patches to the channel dimension
+        batch_size, seq_length, feature_dim = image_features.shape
```

Ruff (0.15.13)

[warning] Unpacked variable `feature_dim` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)